Form/Type/Topic/ReplyType.php

```diff
@@ -26,7 +27,14 @@ public function __construct($loggedIn) {
     public function buildForm(FormBuilderInterface $builder, array $options) {
 
         $builder->add('message', 'textarea', [
```

Comment:
would be wise to convert `'textarea'` to `Symfony\Component\Form\Extension\Core\Type\TextAreaType` and all similar items. this makes them Symfony 3.0 compatible and able to run on Core-2.0 sooner